static/app/utils/replays/replayDataUtils.tsx

```diff
@@ -72,7 +72,7 @@ export function replayTimestamps(
     .map(rawCrumb => rawCrumb.timestamp)
     .filter(Boolean);
   const rawSpanDataFiltered = rawSpanData.filter(
-    ({op}) => op !== 'largest-contentful-paint'
+    ({op}) => op !== 'web-vital' && op !== 'largest-contentful-paint'
```

Comment:
```suggestion
    ({op}) => op !== 'web-vital'
```

looks like we got rid of `largest-contentful-paint` op in static/app/utils/replays/types.tsx, so could we remove the second condition?